object-manipulation-1/src/get-keys.js

```diff
@@ -0,0 +1,15 @@
+/* exported getKeys */
+function getKeys(object) {
+  var keys = [];
+
+  for (var property in object) {
+    keys.push(property);
+  } return keys;
```

Comment:
Please always keep return statements on their own line.

Reply:
fixed! thank you